src/app/shared/links/links-list/links-accordeon.component.html

```diff
@@ -19,6 +19,7 @@
           <links-list-table
             *ngIf="{otherCollection: linkType | linkTypeOtherCollection:(collections$ | async):collection.id} as linkData"
             class="d-block mt-2"
+            [visible]="data.openedGroups.get(linkType.id)"
```

Comment:
Why did you use `Map` ? We use `Record` or simple `{}` .. calling methods in template is not recommended.